usr/share/rear/layout/save/GNU/Linux/230_filesystem_layout.sh

```diff
@@ -231,6 +231,13 @@ fi
                 echo -n " uuid=$uuid label=$label"
                 ;;
             (btrfs)
+                # FIXME: Support for multi-disk BTRFS should be implemented sooner or later
+                # because it is the default layout for multi-disk Fedora Workstation installations.
+                # See: https://github.com/rear/rear/issues/2028
+                if ! grep -q "Total devices 1" <(btrfs filesystem show "$mountpoint"); then
```

Comment:
@jsmeix any chance to have something more robust that matching on the english text output of the tool? E.g. counting lines with `/dev` or something like that?